### ISSUE TITLE
Will there be any update for 2.1 soon?

### ISSUE THREAD

**Issue by developer-lindner** [magento/magento2#6335]
Will there be any update for 2.1 soon which addresses most issues?
Or do we have to wait till 2017?

**Comment by andidhouse** [comment]
Also interested. 

At this point one of the worst things of magento2 is that there is no timeline at all so projects can´t be planned. As magento 2.1 has many major issues a update is really necessary but without any timeline it is really hard to work with magento2 at the moment. 

Is there any plan to change this?

**Comment by GlobalOutsourcing** [comment]
We are expressing interest in this also. We have invested a lot resources and money onto some projects and would prefer to see an update to 2.1 very soon with most issues addressed and resolved.

Thanking you in advance.

**Comment by Krapulat** [comment]
At this time Magento 2 is not for production projects. There are a lot of issues.

**Comment by developer-lindner** [comment]
Tell us something new ;)

**Comment by coredevel** [comment]
OK so we're looking to use Mage2 for a production site and the above comments don't fill me with great confidence in adopting the platform :(  Are we saying Magento 2 isn't ready full stop, or talking about recent minor releases, and is there any documentation about specific issues other than open github issues?

**Comment by developer-lindner** [comment]
@richdel You should read some of them... Without a major update - it's not usable. It's disappointing to have trusted. DON'T USE IT! It's a trap! Seriously...

**Comment by coredevel** [comment]
@developer-lindner thanks for your response. Oh dear :( I've been flicking through some and yes I can see the overall instability of it, man such a great shame.  You're talking major release, you mean wait until Magento 3!?  Afaik there are already a handful of well established and well known companies running with Magento 2, I can only assume they have adopted the Enterprise edition.

**Comment by GlobalOutsourcing** [comment]
@developer-lindner @richdel It's not as bad, we are using it and have developers working on Magento 2.1 site, quite a few extensions and themes which are suitable and have been released. There are issues that need to be addressed, however, I would not say do not use it even though it is a personal decision.

We invested a lot in our Magento 1 site and have decided to migrate to Magento 2.1 a few months back. It has been a challenge as it is a new system but we are happy with it at the moment.

Overall, it would cost us a lot more to migrate from Magento 1 to 2.0 due to the customization etc. therefore decided to go into the latter soon as.

At the end of the day, an update will be released at some point (sooner rather than later) but time frame is not known. Hopefully, the Magento community team are onto it and will provide an update soon, fingers crossed.

In relation to the above, we do appreciate the free platform to establish our product and service which is more than enough at this point of time.

Best of luck!

**Comment by developer-lindner** [comment]
@GlobalOutsourcing So, you're running Magento 2.1 in production?
What about those issues like the broken WYSIWYG-Editor? where directives like {{media url=""}} don't work? 

I also highly appreciate the work they've put into it... just want to get kind of time frame from the magento community team when we'll see major issues fixed.

**Comment by andidhouse** [comment]
I think the main problem is really that all the bugs reported here are put into a "internal ticket system". There are some major bugs around here reported moths ago and not solved. 

Also a big problem i think is that the community has no effect on which bugs need to be fixed first. Think about the mobile picture bug which blocks the magento site to be usable at mobile devices - i think most of the stores online with 2.1 are not even aware this bug does even exist. 

There is no official bug-tracker here (a list or something) and no order in solving the bugs which is in my view very bad. 

I also do not understand the approach of magento. The goal must be to fix ALL major bugs first and then move on to new features. Now they implement new features (which cause new bugs) and just fix "some" bugs with a release.

I also think there is a massive gap in quality testing as many bugs seem to be fixed in a older version and then appear once again in a newer version.

I think M2 will be a great system for ecommerce soon but at the moment i have not seen so much bugs in a eccomerce system at all and we worked with them all...

**Comment by GlobalOutsourcing** [comment]
@developer-lindner Issues that we are coping with at the moment, refreshing the page is not a big deal really and it comes back, in fact it is the first I do when editing something as am use to it until it is released in an update. We are using a plugin that assists us also. It is not dramatically affecting the development to be honest.

We are in development, not in production but my point is that there is no reason why for example us, not use Magento 2 because we have to refresh the page for time being in order to get access to the WYSIWIG editor.

Like yourself, I personally would love to see the next update release and let's face it we will sooner than later as I have stated earlier.

Just don't let it frustrate you as much as it is and give up on Magento 2.

**Comment by developer-lindner** [comment]
@GlobalOutsourcing I don't want to give up and highly appreciate the architecture which lets me learn some cool new stuff. But some people have clients which are already waiting for positive feedback...

**Comment by heldchen** [comment]
yeah, desperately waiting for 2.1.1 as well - cherry picking fixes out of develop is a pain. I really hope magento will start to release bugfix releases faster and more frequently.

**Comment by benmarks** [comment]
Hey folks. We're watching this thread, as we watch all the threads here and on other forums. 

There is a lot of work just underway to deal with issues posted here, as well as developer experience improvements. We're also working out how to provide more visibility for tickets. 

See https://twitter.com/sherrierohde/status/768203639357894656 for a little more info.

**Comment by developer-lindner** [comment]
@benmarks Nice to hear, but could you give us a time frame? improving developer experience (ui components for example) will take some months i guess...  When will we see crucial fixes to Catalog / Attributes / WYSIWYG-Editor?

**Comment by plentyhappy** [comment]
benmarks...I like the improvements...love the feature request voting.

**Comment by hostep** [comment]
@benmarks, @piotrekkaminski, @alankent: https://github.com/magento/magento2/issues/6295#issuecomment-242488388

**Comment by coredevel** [comment]
@benmarks thanks for the info however things seem to feel a little disconnected and makes for a somewhat disparate project management landscape - so there's now GitHub issues, a forum and your own internal PM/ticketing software?

**Comment by coredevel** [comment]
@GlobalOutsourcing thank you for providing your perspective and experience with 2.1, we're ready to begin building a store, even if at this stage to get some development experience with working with the new architecture and we can patch it along the way as and when releases are made.

@andidhouse I couldn't agree more with you re the disconnected ticketing systems, i.e. what's the point in using GitHub if issues are cherry picked and put into an internal ticketing system for the Magento core developers to work on.  It seems to me like there needs to be a synergy between the internal Magento team - visionaries, project managers, developers - and the external open source community to enable true collaboration, allowing developers whether internal or external to contribute and fix issues - at the moment it doesn't seem reflect the true nature of open source development imho or maybe I'm missing something?

**Comment by sivajik34** [comment]
@benmarks it's better to stop implementing new features for 2 months,assign all resources to solving open bugs.so that magento2 will be stable & people get confidence.so everyone will start to working with magento 2.

**Comment by liquidia** [comment]
2.1  release was more about introducing more bugs than fixing exiting one !

**Comment by Cacasapo** [comment]
All in all, my 2.1 experience has been a nightmare, and while I'm not a developer, I am a server admin and have many years in IT. Things got better when I learned to immediately hit github whenever I saw any odd behaviour to check for bugs. I can't see how anyone without extensive troubleshooting skills can be expected to run 2.1 in production as it currently is. 

Just https://github.com/magento/magento2/issues/5929, https://github.com/magento/magento2/issues/5923 and https://github.com/magento/magento2/issues/4785 are serious enough to be show stoppers.

2.1 should never have reached RC, IMO, but I'm hoping 2.1.1 will include as many fixes as possible from the dev branch, bringing M2.1 into a relatively bug-free and stable platform we can use without questioning our own sanity.

**Comment by developer-lindner** [comment]
@Cacasapo @benmarks Looking through most of the issues here the last weeks and months, it seems like a big mess... Catalog, Attributes, Cart, Checkout, Backend... so many broken areas...  I can't imagine that someone really uses it in production. When will we be able to use it for real?!

**Comment by heldchen** [comment]
> When will we be able to use it for real?!

well, I'll know if it's really usable on september 5th - the very very last deadline for our current shop migration project to go online. 

yeah, there are some broken things in 2.x - our company reported lots of issues. but you can work around most of them. we currently carry about 15 local patches and made quite a few modules that provide interceptors to make things work as intended. and lets not forget: 1.x wasn't without its problems.

my real annoyance with 2.x is not really the broken things, but how they are handled. there are lots of bugfixes pushed to the develop branch, but cherry picking them into a "stable", official release is a pain. I really hope magento pushes for faster bugfix releases very soon. 

(yeah I know, with EE comes great support and real patches. but it's currently a very hard sell to a 1.x CE customer, and our EE customers will wait a few dot releases before even starting to think about migration)

**Comment by Cacasapo** [comment]
@developer-lindner  > When will we be able to use it for real?!
As @heldchen mentioned, it will all depend on how the patching is handled for 2.1.1 and future releases. 

I will be going live with the store I'm working on towards the middle or end of September. This is a very small shop with just over 250 products (under 100 configurable products) that runs payments through paypal and only has a few extensions installed along with a theme I made some changes to.

Since it's such a tiny shop and I'm committed, I can work around the bugs I've come across until a stable product is out.

**Comment by stevekem** [comment]
2.1.1 has been released yay!!!

**Comment by developer-lindner** [comment]
@stevekem what's fixed?

**Comment by stevekem** [comment]
@developer-lindner A couple big issues were noted as fixed as well as some smaller ones. I'm hoping that all the fixes were not stated in the release notes and a lot of the smaller bugs reported on here awhile ago were taken care of as well:
http://devdocs.magento.com/guides/v2.1/release-notes/ReleaseNotes2.1.1CE.html

Can anyone confirm if Magento notes every single bugfix in there releases?

**Comment by hostep** [comment]
@stevekem & @developer-lindner: Although Github isn't very good at giving you an overview of big diffs, you can check the difference out here: https://github.com/magento/magento2/compare/2.1.0...2.1.1
But I would strongly advice you to use a git client on your local machine, clone the M2 repo and check the diff using your local git client, it should perform a lot better then the Github diff view.

**Comment by developer-lindner** [comment]
Interesting changes, but no changes to the unusable directives in wysiwyg editor ( {{media url=""}} ) for example.

**Comment by Cacasapo** [comment]
@developer-lindner Yeah, my hopes are dashed.

**Comment by liquidia** [comment]
Atleast 3 issues still exist
1.Missing link to Web setup wizard (when root is pub)  #5235
2.XML File Path issue  (when root us pub)  #2802
3.CMS url rewrite issue #5923

**Comment by andidhouse** [comment]
According to the changelog 6? github issues are solved? 
Am i wrong? 

http://devdocs.magento.com/guides/v2.1/release-notes/ReleaseNotes2.1.1CE.html

Or are more issues solved and not reported? 
I mean here are for example more then 30-40 priority one issues flying around for month blocking every normal shop totally and now there are 6! issues solved?

Or do i misunderstand something here?

If magento goes on like this in my opinion version 2 will be stable in lets say 2018?

**Comment by Bartlomiejsz** [comment]
@benmarks, @piotrekkaminski or someone from Magento team. Could you write here your opinion about suggestion to stop implementing new funcionalities and focus for at least one month (it should be enough in my opinion) on resolving issues from community and merging helpful pr's?

**Comment by benmarks** [comment]
To all: there have been some suggestions that Magento core team should focus on resolving issues and improving developer experience. The timing of these comments is perfect, as this decision was already made by us two weeks ago.

We don't have an explicit duration for this period of focused issue burn down and developer experience improvements, but I suspect that you'll see the effects during the next month or two. 

So, please continue to report tech issues here. Feature requests should be posted in [the appropriate community sub-forum](https://community.magento.com/t5/Magento-2-Feature-Requests-and/idb-p/feature-requests). Tech questions should be directed to Magento Stack Exchange or the community forums as you see fit. 

Also, we are working out improved tracking of internal ticket progress here. It means switching some things up for the engineers, so, give us just a bit of time to get that sorted & in place 🙂

**Comment by hostep** [comment]
@benmarks: since we are now four months further, can you comment on if this is actually happening?
Because I'm seeing a lot of development focus going towards a release of Magento 2.2, instead of a focus for releasing bugfixes for 2.0 or 2.1.

I'm getting a little bit annoyed (well actually quite a lot) by the fact that it has been 3 months since the last bugfix patches (which were only security patches, no problem there, they are more important of course). We would really like to deploy stable Magento 2 shops in the coming weeks/months, but this still feels like a utopia now more then a year after the first "stable" release of Magento 2 because of the fact that there are still a ton of smaller issues all over the place, which mostly have been fixed months ago, but haven't been released in a new version.

I really appreciate those bug fixes are getting done, but next to fixing a bug is also that you have to release this bug fix into a new version, which currently isn't happening at all unfortunately.

Here are some suggestions which could improve in the development of Magento 2:
1) Focus on bug fixes, not on new features
2) Release patch versions more often, smaller releases are so much easier to work with then releasing huge updates which a million things fixed in one go, which nobody can fully understand what is going on (even the M2 team itself I'm afraid)
3) Release on a fixed schedule (like every two weeks would be awesome, I know this sounds controversial and like a lot of work, but this has a lot of upsides for everyone)
4) Be more open as to when a delivered bug fix is going to be released and in which version (maybe use the milestones feature of Github?)

These are just some small suggestions, think about them, in my opinion they make sense...

Regarding the release more versions and much more often (like every two weeks), here are some upsides:

M2 developers can quickly see what has changed between one version and another. They can decided for themselves if they want to upgrade to that particular version because such a release is much more comprehensible.
This is very important, because right now we have to dig through a huge diff between two versions to see what was changed in hundreds and hundreds of files, then decide if we want to upgrade, and if yes, then apply those changes on top of the custom modifications or workarounds we implemented in a specific project to make it work again with the new version.
When the changes are small, this is way more easy for us to see what is happening and will lead to fewer bugs in our own projects. I'm mostly talking about changes in phtml/html/less/css/js/... files (for the php code we often have less problems while upgrading).

The M2 release managers (or whatever they are called), won't have to track a million different commits and figure out which one should be incorporated into which new version on which major releases. If there are smaller releases it will help them because the backlog of unreleased commits is much smaller.

Should a new serious bug get introduced in one of the patch versions, you have a maximum of two weeks to fix this, since a new version is scheduled for then. You can even choose to quickly release a new version in between, should the bug be a critical one or security-related. And the M2 developers would then know this will be fixed in a short amount of time and won't have to figure out a workaround.

Enterprise users (which I'm sure take up a lot of your time, and since they provide you money, this is totally understandable) and the M2 support people for the EE users will benefit from this as well, because they would also have faster bugfixes in new versions.
(I've never worked with the EE version, so I don't know exactly if you are still providing your users with a bunch of patches for all their issues, but going for faster releases could potentially get rid of this patches strategy).

----

I understand all of this can't be changed in a couple of weeks or even months, but just think about it, discuss this internally and try to move M2 forward for the community and for yourselves.

Thanks!

**Comment by heldchen** [comment]
I couldn't agree more. I dread the upcoming (when?) 2.2 release that will introduce 1000s of changes in addition to *maybe* having fixed some of our long outstanding issues like product data corruption.

even for known fixed problems cherry-picking the commits from the dev branch is mostly unfeasible due to the many other changes already introduced.

the current 2.1 version is a nightmare to run in a production system - both for the shop owner as well as for the agency.

**Comment by Ctucker9233** [comment]
@hostep You are right on the mark! If I may add one more thing, @magento-team should also focus on producing fixes for bugs in the branch it was reported in first and not just thrown into the develop branch. I.e. a bug reported in 2.1.0 should be solved for 2.1.0 branch before considering how to apply that fix to develop. That way, we will know that the bug fix is actually compatible. I'm tired of trying to guess or waiting for answered questions on compatibility. In my opinion, each MAGETWO ticket should be released as its own patch.

Also by focusing on multiple branches at once, (2.0.0, 2.10, and soon 2.2.0), the magento team is stretching itself too thin and limiting their ability to fix things.

**Comment by slackerzz** [comment]
@hostep i totally agree with you!
Regarding EE support sadly it's not really different, i have a ticket open from 1 month, they said that since M2 is a complex software they cannot give me any ETA for a solution

**Comment by andidhouse** [comment]
Totally agree with you guys!
We are an agency and focusing on magento2 since over 1.5 years now.
The problem is that we, to make it simple, loose money here every day. In our internal statistics we have about 40-45% of bug fixing time - no way to keep this for us as it is - we never saw an ecommerce platform this buggy to be honest. But the problem for us is not in first step the bugs but how they are handled.

Two examples (out of more then 40! important for us):

We reported a major bug regarding that a product picture is not scrollable on mobile devices over 5 month ago:

https://github.com/magento/magento2/issues/5302
This one is confirmed and also reported here:
https://github.com/magento/magento2/issues/5853

Then the magento team created a fix month later MAGETWO-55608 and reported that the fix fixes the problem. 

Till today the fix is not implemented in the newest magento version (sry - if such a fix takes over half a year to be implemented in a version ... good night!) and also the fix is not complete at all (as reported in the issue. It just solves the problem on the first product picture - second and on product pictures are still bad.

**So for us three problems can be seen here as an example:**
1. Also mayor bugs are not fixed fast but take minimum half a year to be fixed but do net get implemented in any new version
2. The quality control is really bad - fixes are delivered and not tested and this is i think why the community feels that with every release it is getting worse and not better. Only some bugs are fixed - most of them not and a lot of new issues come to the system because of not tested new functions.
3. Nothing changes also magento team says they gonna change the way bugs are handled and treated.

We have not one magento2 live site at the moment - and that after the "stable" version is out for over 1 year now. As an agency we have to make a decision the next month if we support M2 or go for other systems. The problem is:

Clients ask for M2 - our answer is at the moment to have a look at the bugs here and that we can not really recommend M2 at the moment.
Selling these clients 1.x version is also not possible at this moment (buying an outdated system makes no sense for them).

I am really not shure if this goes into the right direction with M2. A lot is promised here the last weeks and moth but nothing happens...

**Comment by benmarks** [comment]
All, hearing you loud & clear. To quote @hostep:  

> I understand all of this can't be changed in a couple of weeks or even months, but just think about it, discuss this internally and try to move M2 forward for the community and for yourselves.

Work along the lines of what has been suggested by multiple people is underway. For elaboration I will defer to a more senior member of our team. I expect he'll respond shortly.

**Comment by Ctucker9233** [comment]
@benmarks Thanks for letting us know someone is listening!

**Comment by woosley-mg** [comment]
Hello all! Jason Woosley here, I run product and engineering for Magento.  First time poster, long time lurker.  Great feedback in this thread, please keep it coming -- it does make a difference. 
 
Let me address what I can so far. While we have been working on the latest patches, we have also been working to improve our ability to release regularly and with more confidence. Specifically, I expect 2.0.11 and 2.1.3 to drop in the next two weeks. The patch is a big one -- somewhere close to 90 issues resolved and a few enhancements. Of course such large releases are harder to integrate and more complex, we will do better going forward. To @hostep remarks, we are already cycling towards smaller patches, but had to dedicate some of our release resources to SUPEE-8788 after issues were reported against prior versions. We'll always index on security first, but I do understand this delay is frustrating. I also agree completely with pre-determined patch dates -- we have some automation work to get done before we can commit to that, but we are striving for it.
 
There are lots (thousands) of successful production stores running the 2.x code base, but I do recognize the defect count is unacceptable, and unnecessarily burdens our partners to either fix it themselves or wait for an indeterminate amount of time for the next release. We are working on being more transparent, trying to get the software in your hands earlier and at higher quality. To that end, we've invested in a new Sustaining Engineering discipline as well as hired a truly impressive operations leader to help drive our automation goals. These new leaders are individuals that I have worked with over a number of years (6+) and have inherent trust in their ability to execute.  Additionally, the work in both of these groups will assist our drive to improve throughput on community pull requests
 
None of this will happen fast enough for me, and I'm sure the same holds true for you, our partners, and for our merchants. I do want to be very clear that we are dedicating an abundance of resources to improving M2 -- there is no M3 in the works, so expect a stable and capable M2 platform that will evolve over time.
 
Thanks for your patience and your feedback -- as @benmarks claims, we are listening and working for you with all of our effort.  I could not be more excited about the future of Magento, and M2 in particular.

**Comment by andidhouse** [comment]
Hi Jason,

many thanks for your open words here first. 

I can give you some feedback here on our experience so for and how in my view this can be improved.

**1. Bugtracking**
To track bugs here over GitHub makes sense - no question. 
But in my opinion this makes only sense if the bugs are really solved in a timeframe and also this timeframe is communicated here. I also realized some bugs are closed here also they are not solved or just part solved.
For now we can report bugs here and - there is no way to track whats happening. The best case you can get here is that a magento team member creates an "internal ticket". And then in most of the cases - nothing happens for a long time. Seems to me that most bugs are reported internal now but are not solved. 

**2. Bug prioritization**

I think bugs need to be prioritized. There are really important bugs ( i will mention two of them at the end of the post for example) out there which really keep us from going live with our shops here.  Other bugs seem to be a "nice to have" but should not be priority one. 
I think at this stage only the really important bugs should be focused and i also think that there should be a way that the community has an influence here which bugs are important and which bugs are not so important.

**3. Timelines**

I think this is often discussed here but there is no solution for it. There is absolutely no official information on what the M2 team is working. Are they working on M2.1.3 or M2.2. - which bugs will be solved here and which bugs not. This is really important to keep us here updated what is happening so we can decide what is the best solution for us and our clients.

**4. Quality of updates**

Also often mentioned here. With every update just a view bugs are solved and new functions keep floating in. Pls do not get me wrong here - we like new stuff! But there is no way to implement new things in my mind when you have reported hundreds of important bugs here. So we "users" feel a little bit confused that new functions are implemented and bugs keep raising with every update.

**5. Quality control**

I am not really shure what kind of quality control is currently implemented in the development but as mentioned before lots of bugs are half fixed and with every update new bugs keep coming in. I think this is quality testing problem and maybe there is a better way to do this.

**6. Suggestion: Publish official test systems of every M2 version**

I am realizing that for lots of bugs it is a ping pong between the M2 team and the reporters here like: which version do you run, updated, downloaded? Which OS are you using? Varnish Redis running? etc. etc.

To make this more simple i really suggest that M2 team is publishing test system with an optimal setup. So users here can for example do videos or screenshots of this system and there is no discussion about OS, versions, etc. etc. - if the bug is present on your own test system - it is a bug and no further communication is necessary.

I think this would make the whole process a lot easier.

Here are for example the two "bugs" i was talking about before which in my view are crucial and "priority one bugs":

**1. Bug**
Ongoing problems with configurable products:
There are so many and massive problems with configurable products. For example no "Out of stock" marker is presented to the customer if for example a size or color is sold out. Imagine an product with 15 sizes or colors you choose over a dropdown. To know which product is available right now you have to click every product in the dropdown and after this ad this to cart. Only then you see if it is sold out or not. This is happening with swatches and dropdown - and really is a blocker for ecommerce sites with colors or dropdowns. There are also related more then 10 bugs reported here relating configurable products. For now the system is really hard to use and can´t be recommended to any client for a live site.

**2. Bug**

I reported i think 5 month ago that on some mobile devices the fotorama script is not working as it blocks the scroll down on the product detail page. This is a major issue as this is really blocking people to buy mobile in M2 stores. This was over 5 month ago and also reported several times after here. If you download now 2.1.2 - the same bug is still present.
I think it should not be that such major bugs do not come into a new release version half a year later. Also a fix is presented here people are reporting here that the fix is not complete working. Also the fotorama script on mobile devices is very slow ( i reported here as well and others proved) - this is also very important for user experience and SEO - there is no solution for this as far as i know.

Really sry for the long text but we as an agency really struggle at the moment with M2 and really hope it changed now regarding "major" bugs.

André

**Comment by timstoel** [comment]
We've recently moved to Magento from another platform and it has been a nightmare.  We're in production but it's terrible and we have so many workarounds and restrictions on use.  There should be a huge disclaimer on the Magento homepage next to the Magento 2 downloads saying this is NOT PRODUCTION READY -- DO NOT USE.

**Comment by ghost** [comment]
@timstoel welcome to the club. Best advice I can offer, hire a part time dev to keep you stable.

**Comment by ghost** [comment]
So who dropped the ball on the 2.1.3. Counterproductive to release such an issue filled update to the public.

**Comment by plentyhappy** [comment]
I have to agree - we are on 2.1.2...did not upgrade to 2.1.3 just because I'm used to all the bugs...did not want to start from square one.  I wish I had just used 1.9x...I have spent 17K on developers (the one I am using now is top notch but the money spent is no fun...these are not even improvements on the site...just bug fixes for the most part) since Feb 2016...all of this being said everyone is super nice at Magento on Github but upper management needs a change.  

Mainly checkout has not worked...it does but constantly chasing problems...it is unbelievable.  My advice is fix the super basic bugs first - please stop adding nice to have features...just fix the most basic bugs.  Do this for a year and then add the cool stuff...back to reality...bugs.

**Comment by ghost** [comment]
http://webshopapps.com/blog/2017/01/what-magento-should-spend-some-of-that-250m-on/

One of magento's gold partners posted this. Sums up how we all feel, hints that magento may move on from CE. Would be nice if a senior member gave us an idea on what's really going on behind closed doors.

**Comment by PaperDS** [comment]
Way to go. Let's just close this issue. 
Where can us users get some clarification on what's going on?

**Comment by acharrex** [comment]
@maksek : Please reopen this ticket!
We have the right to get some clarification!

**Comment by acharrex** [comment]
https://community.magento.com/t5/Just-Ask-Alan/Alan-When-will-there-be-a-really-stable-version-of-Magento2/m-p/56617#U56617

**Comment by ghost** [comment]
Seems they are closing out the same community they use to use as a sale point for the platform.
